src/main/kotlin/com/team4099/robot2022/config/ControlBoard.kt

```diff
@@ -21,4 +21,9 @@ object ControlBoard {
     get() = driver.rightXAxis
 
   val resetGyro = Trigger { driver.startButton && driver.selectButton }
+
+  val runIntakeIn = Trigger { operator.aButton }
+  val putIntakeUp = Trigger { operator.bButton }
```

Comment:
the control scheme this year probably should be different because its really fast/lightweight to lift the intake up. so it probably would be better if when the button was held the intake went down and spun inwards, then when released it automatically retracted up